frontend/ui/data/creoptions.lua

```diff
@@ -528,6 +519,39 @@ Note that your selected font size is not affected by this setting.]]),
                     value_step = 1,
                 },
             },
+            {
+                name = "font_base_weight",
+                name_text = _("Font Weight"),
+                toggle = { "-1", "-½", "0", "+½", "+1", "+1½", "+2" },
+                values = { -1, -0.5, 0, 0.5, 1, 1.5, 2 },
+                args = { -1, -0.5, 0, 0.5, 1, 1.5, 2 },
+                default_value = 0,
+                event = "SetFontBaseWeight",
+                more_options = true,
+                more_options_param = {
+                    value_min = -3,
+                    value_max = 5.5,
+                    value_step = 0.25,
+                    precision = "%+.2f",
+                    value_hold_step = 1,
+                },
+                help_text = _([[Set the font weight delta from "regular" to apply to all fonts.
+
+- 0 will use the "regular (400)" variation of a font.
+- 1 will use the "medium (500)" variation of a font if available
+- 3 will use the "bold (700)" variation of a font if available
+If a font variation is not available, and for fractional deltas, a font will be synthesized from the nearest available weight of the font.]]),
```

Comment:
`as well as for fractional deltas`

Reply:
Could also possibly keep the `variation` thing going, since that's going to be more and more commonly used with the advent of variable fonts, so, `a font variation will be synthesized`

Reply:
Maybe just a `fractional adjustment`?